### DISCUSSION TITLE
How to add XMR via config (NOT using docker)

### DISCUSSION THREAD

**Discussion by camalolo** [btcpayserver/btcpayserver#2381] [Technical Support] [answered]
Hello, sorry if I missed the answer to this in the documentation, but I could not find it :

How can I enable XMR/altcoins in the repository I cloned from https://github.com/btcpayserver/btcpayserver ?

I could not find the details on what parameters to set in $HOME/.btcpayserver/Main/settings.config but I did find how to enable altcoins by guessing the config entry from the source code; however then I got an error message saying that this version of btcpayserver does not have altcoins enabled.

I understand that the recommended approach is to use docker, however, I would really like to know how to configure btcpayserver WITHOUT using the docker version, as I don't need most of the stuff that get installed with the docker system.

I just want to use btcpayserver from this repository.

Thank you !

**Comment by Loosetooth** [comment]
Bump! You're not the only one trying to do this. I'm going to check the config file you mentioned.

Edit: just checked my config file from the extended manual install tutorial. It only contains database settings. Is there any chance that you could share the config settings that you tried in order to enable XMR? And if you would really like to help out, maybe a link to the source code where you got inspiration from to change those settings.

Thanks!

**Comment by Kukks** [comment, marked as answer]
BTCPay runs in Bitcoin only mode by default.  The `run.ps1` and `run.sh` are both configured to use that mode.
You can run in altcoins mode by running the following: `dotnet run --no-launch-profile --no-build -c Altcoins-Release -p .\BTCPayServer\BTCPayServer.csproj -- $args
`

**Reply by Kukks** [reply]
In terms of monero specific config, you can refer to https://github.com/btcpayserver/btcpayserver-docker/blob/master/docker-compose-generator/docker-fragments/monero.yml to see what you need to configure

**Reply by Loosetooth** [reply]
Thanks for the quick response. This already helped me a lot!

Another question: in the .yml file you mention, there's this argument provided when running the monero node: 
`--block-notify="/bin/sh ./scripts/notifier.sh ...`
 And similar for the monero wallet:
`--tx-notify="/bin/sh ./scripts/notifier.sh ...`
Where can we find these scripts? I did a quick search in the btcpay and the btcpay-docker repositories, but could not find anything.

Edit: found it: https://github.com/Kukks/monero-docker/blob/master/scripts/notifier.sh
